facebook/index.html

```diff
@@ -409,11 +409,15 @@ <h2 class="page-title">Facebook Services</h2>
                 <div
                   style="color: #50565b"
                   class="service-title gcard-value--facebook gcard-value--facebook-likes"
-                ></div>
+                >
+                  $1.6 per 1k
+                </div>
                 <div
-                  style="color: #fff"
+                  style="color: #fff; margin-top: 12px"

```

Comment:
Please remove inline styling. Add a CSS class on this and add styles to the CSS class.